### ISSUE TITLE
CKEditorWidget is not compatible with SplitArrayField

### ISSUE THREAD

**Issue by vishalpathak24** [django-ckeditor/django-ckeditor#408]
When we try to use CKEditorWidget with SplitArrayField only TextArea is rendered. 

For Example:
For form field:
`options = SplitArrayField(forms.CharField(widget = CKEditorWidget()),size=4)`

This will result in rendering TextArea boxes with no other attributes.

**Solution Used :**
I have traced back the issue and currently created a custom widget extending CKEditorWidget and finding required fields

```
class SplitArrayCKEditorWidget(CKEditorWidget):
	#Custom Widget which overrides get_context as
	#CKEDITOR Widget does not by SplitArray utilizes it
	
	def __init__(self,*args,**kwargs):
		super(SplitArrayCKEditorWidget,self).__init__(*args,**kwargs)
		self.template_name = 'ckeditor/split_array_widget.html'

	def get_context(self, name, value, attrs):
		json_encode = LazyEncoder().encode
		context = {}
		if value is None:
			value = ''
		final_attrs = self.build_attrs(self.attrs, attrs, name=name)
		self._set_config()
		external_plugin_resources = [[force_text(a), force_text(b), force_text(c)]
                                     for a, b, c in self.external_plugin_resources]

		context['widget'] = {
			'final_attrs': flatatt(final_attrs),
			'value': conditional_escape(force_text(value)),
			'id': final_attrs['id'],
			'config': json_encode(self.config),
			'template_name':self.template_name,
                        'external_plugin_resources': json_encode(external_plugin_resources)
		}
		return context`
```

**Note** that this also needs custom template as following
```

{% load static %}
<div class="django-ckeditor-widget" data-field-id="{{widget.id}}" style="display: inline-block;">
    <textarea{{ widget.final_attrs }} data-processed="0" data-config="{{ widget.config }}" data-external-plugin-resources="{{ widget.external_plugin_resources }}" data-id="{{widget.id}}" data-type="ckeditortype">{{ widget.value }}</textarea>
</div>


```
Basically with _widget_ ahead all the attributes of the source template.
I understand this is raw hack, but might help others and can be corrected in future release.
